app/controllers/rooms_controller.rb

```diff
@@ -13,6 +13,10 @@ def index
     end
   end
 
+  def show
+    @tracks = @room.tracks
+  end
+
```

Comment:
Hi Lea :) can you also consider this scenario. For example, the user saved the url of a specific room _http://localhost:3000/rooms/10_. Then, this room was subsequently deleted, the page will results as follows :) 

![image](https://user-images.githubusercontent.com/79251819/134142472-e502b246-c0fd-4b0d-bea4-eb8000bde5ae.png)